go/engine/pgp_export_key.go

```diff
@@ -166,6 +181,49 @@ func (e *PGPKeyExportEngine) exportSecret(ctx *Context) error {
 	return nil
 }
 
+func getPGPPassphrase(g *libkb.GlobalContext, ui libkb.SecretUI) (keybase1.GetPassphraseRes, error) {
```

Comment:
change to `getPGPExportPassphrase`.  We don't want people using it for any other reason.